src/RefractAPI.cc

```diff
@@ -100,6 +116,26 @@ namespace drafter {
 
     refract::IElement* DataStructureToRefract(const snowcrash::DataStructure& dataStructure, bool expand)
     {
+       refract::IElement* msonElement = MSONToRefract(MakeSectionInfoWithoutSourceMap(dataStructure));
```

Comment:
Looks like you are not using this function anywhere. Please remove this whole function.